### ISSUE TITLE
How is validation split created?

### ISSUE THREAD

**Issue by JiankunW** [mwray/Joint-Part-of-Speech-Embeddings#4]
1. The validation split is used for evaluation in your repo. But how is validation split created? I could not find the description about it in your paper.

A problem from the val split is that the annotation dataframes between your [provided](https://github.com/mwray/Joint-Part-of-Speech-Embeddings#data) train and val split have overlapped videos.
```python
>>> tr_annos_jpose = pd.read_pickle(os.path.join(root_path, 'data/dataframes/EPIC_100_retrieval_{}.pkl'.format("train")))
>>> va_annos_jpose = pd.read_pickle(os.path.join(root_path, 'data/dataframes/EPIC_100_retrieval_{}.pkl'.format("validation")))
>>> print(len(tr_annos_jpose), len(va_annos_jpose))
67219 4834
>>> set(va_annos_jpose.index) - set(tr_annos_jpose.index)
{'P22_17_238'}
```

2. The other question is that why the number of videos in the train split in this repo (`67219` as shown above) is different with that in [epic-kitchens-100-annotations](https://github.com/epic-kitchens/epic-kitchens-100-annotations) repo (`67217`).

Jiankun

**Comment by mwray** [comment]
Hi Jiankun,

Thanks for your questions and interest in the code.

1. Due to the dataset being used for other challenges, initially there was no validation split for the retrieval task as the validation split for EPIC-Kitchens-100 is used for the test set for the challenge due to potential leakage of information. When releasing the code I thought it would still be best to be able to run the code and check that everything is set up correctly/show that the model gets the results we publish on the README without having to submit to the challenge which is why there is the non-traditional validation set.

2. Thank you for raising this, I will look into this further, but for now, please use the annotations in the epic-kitchens-100-annotations repo.

Kind Regards,

Michael
